src/core/api/link_raw_api.cpp

```diff
@@ -357,13 +194,12 @@ void LinkRaw::InvokeReceiveDone(otRadioFrame *aFrame, otError aError)
         if (aError == OT_ERROR_NONE)
         {
             otLogInfoPlat(&mInstance, "LinkRaw Invoke Receive Done (%d bytes)", aFrame->mLength);
+            mReceiveDoneCallback(&mInstance, aFrame, aError);
```

Comment:
We have a bunch of stats/counters in the MAC to record error occurrences.  If we do not want to pass all the errors to host, we'll need to consider adding the counters into the radio.

Reply:
I'm also considering merging LinkRaw with Mac to reduce duplicated code after this feature passes all tests, which should take some additional efforts. Shall we deal with the counters later or in this PR?

Reply:
I'm fine either way, what do you prefer?

Reply:
I would prefer do it later, so that this feature could be available earlier.